diff -burN csmash-0.6.6.orig/loadparts.cpp csmash-0.6.6/loadparts.cpp
--- csmash-0.6.6.orig/loadparts.cpp	2003-11-30 18:38:43.000000000 +0100
+++ csmash-0.6.6/loadparts.cpp	2003-11-30 18:40:20.000000000 +0100
@@ -245,7 +245,7 @@
 
 	while ('\\' == line[l-1]) {
             // concat next line(s)
-	    int bufsize = clamp(0U, sizeof(line)-l, sizeof(line)-1);
+	    int bufsize = clamp((size_t)0U, sizeof(line)-l, sizeof(line)-1);
 	    fgets(&line[l-2], bufsize, fp);
 	    if (feof((FILE*)fp)) break;
 	    l = strlen(line);
